diraccfg/__main__.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import argparse
+import json
+import os
+import sys
+import re
+
+from .cfg import CFG
+
+
+def parseArgs():
+  parser = argparse.ArgumentParser(description='Parser for DIRAC cfg files')
+  subparsers = parser.add_subparsers(help='Actions to run with the ')
+
+  parserJson = subparsers.add_parser('as-json', help='Dump the entire configuration as JSON')
+  parserJson.add_argument('cfgfile')
+  parserJson.set_defaults(func=lambda a: dumpAsJson(a.cfgfile))
+
+  parserSort = subparsers.add_parser('sort-versions', help='Read JSON from stdin and prints JSON list of sorted version numbers')
+  parserSort.add_argument('--allow-pre-releases', action='store_true')
+  parserSort.set_defaults(func=lambda a: sortVersions(a.allow_pre_releases))
+
+  args = parser.parse_args()
+  args.func(args)
+
+
+def dumpAsJson(cfgFilename):
+  if not os.path.isfile(cfgFilename):
+    sys.stderr.write('ERROR: %s does not exist\n' % cfgFilename)
+    sys.exit(1)
+  res = CFG().loadFromFile(cfgFilename)
+  print(json.dumps(res.getAsDict()))
```

Comment:
I didn't try with a dirac cfg file that contains comments, as in that case they should be remove.